exampleSite/config.toml

```diff
@@ -28,6 +28,7 @@ pygmentsCodefencesGuessSyntax = true
   socialShare = true
   delayDisqus = true
   showRelatedPosts = true
+  mainSections = "post"
```

Comment:
It is not needed to enter mainSections in the config. Also the way you have entered this parameter is wrong since it is an array not a string.

Please remove this.